arch/arm/src/nrf52/hardware/nrf52_spi.h

```diff
@@ -153,13 +154,13 @@
 #define SPIM_ENABLE_DIS             (0)        /* Disable SPIM */
 #define SPIM_ENABLE_EN              (0x7 << 0) /* Enable SPIM */
 
-/* PSEL(MOSI/MISO/SCK/CSN) Register */
+/* PSEL* Registers */
 
-#define SPIM_PSEL_PIN_SHIFT         (0)       /* Bits 0-4: pin number */
-#define SPIM_PSEL_PIN_MASK          (0x1f << SPIM_PSEL_PIN_SHIFT)
-#define SPIM_PSEL_PORT_SHIFT        (5)       /* Bit 5: port number */
-#define SPIM_PSEL_PORT_MASK         (0x1 << SPIM_PSEL_PORT_SHIFT)
-#define SPIM_PSEL_CONNECTED         (1 << 31) /* Bit 31: Connection */
+#define SPIM_PSEL_PIN_SHIFT         (0)       /* Bits 0-4: SCK pin number */
+#define SPIM_PSEL_PIN_MASK          (0x1f << SPIM_PSELSCK_PIN_SHIFT)
+#define SPIM_PSEL_PORT_SHIFT        (5)       /* Bit 5: SCK port number */
+#define SPIM_PSEL_PORT_MASK         (0x1 << SPIM_PSELSCK_PORT_SHIFT)
+#define SPIM_PSEL_DISCONNECTED      (1 << 31) /* Bit 31: Disconnect */
```

Comment:
This bit is called CONNECT in the manual, not DISCONNECT. Shouldn't we stick to the manufacture's name?

Reply:
The problem is that it was "connect" with value 1, which is opposite of what it does (0 means connected), so to actually provide the expected meaning the definition should be for bit 1, which applies for "disconnect". I can change the comment if you prefer.

Reply:
Yes I agree it is confusing. But all other peripherals use CONNECT in PSEL definitions.
So we can decide to update all other headers or leave it unchanged.
If we decide to change, I think it should be in a separate PR

Reply:
Ok, I'll remove the change. 
These PSEL definitions are the always same so we could actually do it in a single header and include it where needed.

Reply:
Yes, that is a good idea. We can also move all the functionality related to the PSEL configuration to one place. Kind of like `nrf52_spi_pselinit()` but common to all peripherals.